keras_trainer/trainer.py

```diff
@@ -19,6 +20,7 @@ class Trainer(object):
         'sgd_lr': {'type': float, 'default': 0.01},
         'pooling': {'type': str, 'default': 'avg'},
         'weights': {'type': str, 'default': 'imagenet'},
+        'num_gpu': {'type': int, 'default': 1},
```

Comment:
Small nitpick: can you rename this to `num_gpus`? 😄

Reply:
Also `'default': None`

Reply:
yes, sure!